nx-dev/nx-dev/redirect-rules.js

```diff
@@ -1125,6 +1125,12 @@ const blogPosts = {
     '/blog/nx-cloud-3-0-faster-more-efficient-modernized',
 };
 
+const featurePagesUpdate = {
+  '/nx-cloud/troubleshooting/explain-with-ai':
+    '/nx-cloud/features/explain-with-ai',
+  '/nx-cloud/features/ai-features': '/nx-cloud/concepts/ai-features',
+};
```

Comment:
When you create a redirect rule, you should also update all existing links this rule is taking care of, or it will continue to be used.

Reply:
right, doing that in a bit